src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py

```diff
@@ -172,7 +172,7 @@ def __call__(
             self.unet.in_channels,
         )
         if latents is None:
-            latents = jax.random.normal(prng_seed, shape=latents_shape, dtype=self.dtype)
+            latents = jax.random.normal(prng_seed, shape=latents_shape, dtype=jnp.float32)
```

Comment:
can we use the models `dtype` here ?

Reply:
`self.dtype` didn't work for me when using `bfloat16`. The latents generated in the denoising loop are always computed in `float32`, so they don't match the previous type if it was something else.